Build macsio with hdf5 1.13

mchaarawi opened the pull request:
PR-repos-el7: hdf5@PR-36 hdf5-vol-daos@PR-32
PR-repos-el8: hdf5@PR-36 hdf5-vol-daos@PR-32
PR-repos-leap15: hdf5@PR-36 hdf5-vol-daos@PR-32

Signed-off-by: Mohamad Chaarawi <mohamad.chaarawi@intel.com>

mchaarawi committed:
Build macsio with hdf5 1.13

- also update packaging

PR-repos-el7: hdf5@PR-36 hdf5-vol-daos@PR-32
PR-repos-el8: hdf5@PR-36 hdf5-vol-daos@PR-32
PR-repos-leap15: hdf5@PR-36 hdf5-vol-daos@PR-32

Signed-off-by: Mohamad Chaarawi <mohamad.chaarawi@intel.com>

brianjmurrell commented:
Is this needing rebuilding because hdf5 is (unnecessarily?  if they really did bump it without any actual abi change, we really need to have a conversation with them about this practice) bumping their library major version?

If so, you need to update the specfile here to bump the `Release:` to ensure that when this lands it does not overwrite a macsio with the previous hdf5 library major version.

mchaarawi commented:
> Is this needing rebuilding because hdf5 is (unnecessarily? if they really did bump it without any actual abi change, we really need to have a conversation with them about this practice) bumping their library major version?
> 
> If so, you need to update the specfile here to bump the `Release:` to ensure that when this lands it does not overwrite a macsio with the previous hdf5 library major version.

yes indeed. i will bump the Release:

mchaarawi committed:
update Release since we are using a new HDF5 1.13.1 verison

Signed-off-by: Mohamad Chaarawi <mohamad.chaarawi@intel.com>

mchaarawi committed:
rebuild with correct repos

PR-repos-el7: hdf5@PR-36 hdf5-vol-daos@PR-32
PR-repos-el8: hdf5@PR-36 hdf5-vol-daos@PR-32
PR-repos-leap15: hdf5@PR-36 hdf5-vol-daos@PR-32

Signed-off-by: Mohamad Chaarawi <mohamad.chaarawi@intel.com>
